modules/oadp-s3-and-gov-cloud.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="oadp-s3-and-gov-cloud_{context}"]
+= About Amazon Simple Storage Service, Identity and Access Management, and GovCloud
+
+Amazon Simple Storage Service (Amazon S3) is Amazon's storage solution for the internet. As an authorized user, you can use this service to store and retrieve any amount of data whenever you want, from anywhere on the web.
```

Comment:
Do not use ’s to show the possessive form of abbreviations or product names.
```suggestion
Amazon Simple Storage Service (Amazon S3) is a storage solution of Amazon for the internet. As an authorized user, you can use this service to store and retrieve any amount of data whenever you want, from anywhere on the web.
```

> Do not use ’s to show the possessive form of abbreviations or product names.
Incorrect example: IBM Instana’s capabilities yield real-time monitoring, proactive issue diagnosis, and performance optimization.
Correct example : The capabilities of IBM Instana yield real-time monitoring, proactive issue diagnosis, and performance optimization.